### ISSUE TITLE
Donut 2 NW and north side of station consistently do not receive power

### ISSUE THREAD

**Issue by FlameArrow57** [goonstation/goonstation#14027]
### Labels

[BUG]

### Description

On Donut 2, APCs will continually discharge and lose power on the NW and north side of the station

### Steps to reproduce

1. Seeable from round start

### Expected Behavior

For them to receive at least some charge

### Additional Information & Screenshots

I think it's because there's a separate SMES for the north and south sides of the station maybe? Regardless, checking wires on the south side of the station with a multitool, it has a small power applied to it with no singulo set up, whereas the north side of the station receives none roundstart, causing QM to lose power quickly.

Even if this is intended, it feels unintuitive maybe

**Comment by M-Earthfire** [comment]
Iirc, Donut 2 got 4 seperated powernetworks, each powered by 1 SMES from the engine-side.

**Comment by BatElite** [comment]
Correct, I'm guessing the north side struggles because of cargo. Engineers seemingly don't know how the map's powernets are set up and only consistently configure the south SMES. There's a note about it in engineering itself I'm pretty sure.

**Comment by pgmzeta** [comment]
Not speaking on intent, but the reason is that the north side SMES Output doesn't start enabled, so even if power is set up there is zero charge going through.
